resources/public/api/conf/1.0/endpoints/createPeriodicUpdatePeriod.raml

```diff
@@ -0,0 +1,46 @@
+is:
+  - headers.acceptJson
+  - headers.testScenarioHeader
+  - headers.contentTypeJson
+  - errors.formatNino
+  - errors.formatBusinessId
+  - errors.formatFromDate
+  - errors.formatToDate
+  - errors.formatValue
+  - errors.incorrectOrEmptyBody
+  - errors.ruleBothExpensesSupplied
+  - errors.ruleToDateBeforeFromDate
+  - errors.ruleOverlappingPeriod
+  - errors.ruleMisalignedPeriod
+  - errors.ruleNotContiguousPeriod
+  - errors.ruleNotAllowedConsolidatedExpenses
+  - errors.clientOrAgentNotAuthorised
+  - errors.notFound
+displayName: Create a self-employment periodic update [test only]
```

Comment:
Would be best to capitalise Self-Employment Periodic Update to match with the above annual summary endpoint, keep it consistent